src/Altinn.AccessManagement.Core/Configuration/GeneralSettings.cs

```diff
@@ -19,5 +19,15 @@ public class GeneralSettings
         /// Option to disable csrf check
         /// </summary>
         public bool DisableCsrfCheck { get; set; }
+
+        /// <summary>
+        /// Gets or sets the RuntimeCookieName
+        /// </summary>
+        public string RuntimeCookieName { get; set; }
```

Comment:
RuntimeCookieName is currently not set by the GeneralSettings section i appsettings.json but is set by JwtCookieName in PlatformSettings.

Reply:
Right, I see that it was taken in with a merge conflict resolve :) I have removed it from general settings